sema/check/check.py

```
@@ -0,0 +1,133 @@
+    for file in yaml_files:
+        log.debug(f"Loading {file}")
+        try:
+            with open(os.path.join(str(input_folder), file), "r") as stream:
+                try:
+                    data = yaml.safe_load(stream)
+                    if data:
+                        if isinstance(data, list):
+                            rules.extend(data)
+                        else:
+                            rules.append(data)
+                except Exception:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)